numberFormatting.js

```diff
@@ -0,0 +1,3 @@
+var numberFormat = (number) => {
```

Comment:
Why did you use the var keyword when you could have used let? Additionally, since I noticed that your variable never changed, it would be more appropriate to use const for better code clarity and immutability.